src/System.Management.Automation/DscSupport/CimDSCParser.cs

```diff
@@ -633,7 +633,7 @@ public static void Initialize(Collection<Exception> errors, List<string> moduleP
         {
             s_tracer.WriteLine("Initializing DSC class cache force={0}");
 
-            if (System.Management.Automation.Platform.IsLinux)
+            if (System.Management.Automation.Platform.IsLinux || System.Management.Automation.Platform.IsMacOS)
```

Comment:
nit: can we remove the `System.Management.Automation.` parts?

Reply:
sure